<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NaujasEka</class>
 <widget class="QDialog" name="NaujasEka">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_model">
       <property name="text">
        <string>EKA modelis</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_model">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_ekanr">
       <property name="text">
        <string>EKA korpuso numeris</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_ekanr">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_cert">
       <property name="text">
        <string>Paso numeris</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_cert">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_number">
       <property name="text">
        <string>Naudototas kartu</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_number">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="maxLength">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_user">
       <property name="text">
        <string>Vartotojas</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_user"/>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_ekaplace">
       <property name="text">
        <string>Stovėjimo adresas</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_ekaplace">
       <property name="maxLength">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="1">
      <widget class="QCheckBox" name="checkBox_gar">
       <property name="text">
        <string>Garantija</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QDateEdit" name="dateEdit_nuom"/>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="checkBox_place">
       <property name="text">
        <string>Miestas</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="checkBox_nuom">
       <property name="text">
        <string>Nuoma</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QDateEdit" name="dateEdit_gar"/>
     </item>
     <item row="1" column="2">
      <widget class="QDateEdit" name="dateEdit_prof"/>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_vietove">
       <property name="text">
        <string>Vietovė</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_registruota">
       <property name="text">
        <string>Registruota</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_profilaktika">
       <property name="text">
        <string>Sutartis</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QDateEdit" name="dateEdit_reg"/>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>31</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="3">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_save">
       <property name="text">
        <string>Išsaugoti</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_close">
       <property name="text">
        <string>Uždaryti</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>tableView</zorder>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>NaujasEka</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>679</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>761</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
